src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/ImageIndexEditorTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.ComponentModel;
+using System.Drawing;
+using System.Drawing.Design;
+using Moq;
+
+namespace System.Windows.Forms.Design;
+
+public class ImageIndexEditorTests
+{
+    private readonly ImageIndexEditor _editor = new();
+    private ITypeDescriptorContext? _context = new Mock<ITypeDescriptorContext>().Object;
+
+    [Theory]
+    [BoolData]
+    public void GetPaintValueSupported_WhenContextIsNullOrNot_ReturnsTrue(bool hasContext)
+    {
+        _context = hasContext ? _context : null;
+
+        bool result = _editor.GetPaintValueSupported(_context);
+
+        result.Should().BeTrue();
+    }
+
+    [Theory]
+    [InlineData(null)]
+    [InlineData("Text")]
+    [InlineData(1)]
+    public void PaintValue_WithStringOrIntOrNull_DoesNotThrow(object? value)
+    {
+        using var bitmap = new Bitmap(10, 10);
```

Comment:
Let's use target type new
```suggestion
        using Bitmap bitmap = new(10, 10);
```